server/routes/offenceCodeDecisions/offenceCodeDecisions.ts

```diff
@@ -0,0 +1,84 @@
+import { Request, Response } from 'express'
+import url from 'url'
+import decisionTrees from '../../offenceCodeDecisions/DecisionTrees'
+import { FormError } from '../../@types/template'
+import validateForm from './offenceCodeDecisionsValidation'
+import PlaceOnReportService from '../../services/placeOnReportService'
+
+import type IncidentRole from '../../offenceCodeDecisions/IncidentRoles'
+import { properCaseName } from '../../utils/utils'
+
+type PageData = {
+  error?: FormError
+  selectedDecisionId?: string
+}
+
+export default class OffenceCodeRoutes {
+  constructor(private readonly placeOnReportService: PlaceOnReportService) {}
+
+  private async placeholderValues(adjudicationNumber: string, res: Response) {
```

Comment:
This should maybe go in a service - but I have a suspicion that we'll need to also grab stuff off the session - so maybe a helper even.